src/Symfony/Component/Uid/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.3.0
+-----
+
+ * added commands to generate and inspect uuid and ulid
```

Comment:
```suggestion
5.3
---

 * Add commands to generate and inspect Uuid's and Ulid's
```

Fabien documented how to write `CHANGELOG.md` files here, please keep a look and adjust it accordingly.

https://github.com/symfony/symfony-docs/pull/14830

Thanks 🙏